src/backend/joanie/tests/core/api/admin/test_skills.py

```diff
@@ -0,0 +1,199 @@
+"""Tests for the Skill Admin API endpoints."""
+
+from http import HTTPStatus
+
+from joanie.core import factories, models
+from joanie.tests.base import BaseAPITestCase
+
+
+class SkillAdminApiTest(BaseAPITestCase):
+    """Test for the Admin SkillViewSet."""
+
```

Comment:
Where is the test that checks the method `validate` where we raise `ValidationError`  in the `AdminSkillSerializer` ?
I feel like it's missing

Reply:
This is dev residue as initially I planned to create a nested product route but actually it was useless, I abort this dev.